apps/Cargo.toml

```diff
@@ -74,8 +74,7 @@ blake2b-rs = "0.2.0"
 borsh = "0.9.0"
 byte-unit = "4.0.13"
 byteorder = "1.4.2"
-# https://github.com/clap-rs/clap/issues/1037
-clap = {git = "https://github.com/clap-rs/clap/", tag = "v3.0.0-beta.2", default-features = false, features = ["std", "suggestions", "color", "cargo"]}
```

Comment:
@tzemanovic Should we re-enable some of the features? https://docs.rs/clap/latest/clap/_features/index.html

Reply:
yeah, let's re-enable all - we can skip "std" which is a placeholder

Reply:
Actually, the features "std", "suggestions", "color" are default, which are enabled unless disabled by an explicit `default-features = false`. Do we want to disable other default features? Is the "cargo" feature useful for us?

Reply:
alright, the defaults seem fine. I don't think we're using anything that needs "cargo"

Reply:
Good, thanks!